swagger-doclet/src/main/java/com/tenxerconsulting/swagger/doclet/parser/ParserHelper.java

```diff
@@ -860,10 +927,14 @@ public static String paramTypeOf(boolean multipart, Parameter parameter, DocletO
         AnnotationParser p = new AnnotationParser(parameter, options);
         if (p.isAnnotatedBy(JAX_RS_PATH_PARAM)) {
             return "path";
+        } else if (p.isAnnotatedBy(SPRING_MVC_PATH_VARIABLE)) { 
```

Comment:
I think the common functionality of JAXRS and SpringMVC annotations would be better expressed by combining these conditionals.